src/build/env.py

```diff
@@ -18,8 +18,27 @@ class IsolatedEnvironment(object):
     Non-standard paths injected directly to sys.path still be passed to the environment.
     '''
 
-    def __init__(self):  # type: () -> None
+    MANIPULATE_PATHS = ('purelib', 'platlib')
+
+    def __init__(self, remove_paths):  # type: (Collection[str]) -> None
```

Comment:
Maybe change from `Collection` to `Iterable` as `Collection` is not available in Python 3.5.